collector/exec_bsd.go

```diff
@@ -72,6 +75,12 @@ func NewExecCollector() (Collector, error) {
 				description: "Number of fork() calls since system boot.  Resets at architeture unsigned integer.",
 				mib:         "vm.stats.vm.v_forks",
 			},
+			{
+				name:        "boot_time",
```

Comment:
By our convention, metric names should include the unit, so this will become `node_boot_timestamp_seconds`. Either we keep consistency with linux for now and break them all together in a later release, or you could choose to already start using the correct name here.